ext/mixed/display-templates.html

```diff
@@ -77,5 +77,6 @@
 
 <template id="tag-template"><span class="tag"><span class="tag-inner"></span></span></template>
 <template id="tag-frequency-template"><span class="tag" data-category="frequency"><span class="tag-inner"><span class="term-frequency-dictionary-name"></span><span class="term-frequency-separator"></span><span class="term-frequency-value"></span></span></template>
+<template id="tag-search-template"><span class="tag" data-category="search"></span></template>
```

Comment:
Could be useful to abstract away `tag-inner` and add a separator as well for scanning so that it doesn't have to be handled in `DisplayGenerator` for each tag type

Reply:
Separator because while double click is delimited by the block element, scanning isn't. Also https://github.com/FooSoft/yomichan/pull/286